content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,261 @@
+capacities to all vehicles of that type.
+
+To configure, navigate to :menuselection:`Fleet app --> Configuration --> Models`.
+
+.. seealso::
+   :doc:`Create car model <../../../../hr/fleet/models>`
+
+From the :guilabel:`Models` list, select an existing model or click :guilabel:`New` in the top-left
+corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
+vehicle category.
+
+.. _inventory/shipping_receiving/inventory-features:
+
+Inventory settings
+------------------
+
+Next, go to :menuselection:`Inventory app --> Configuration --> Settings` and enable the required
+features for dispatch management.
+
+In the :guilabel:`Operations` section, tick the :guilabel:`Batch, Wave & Cluster Transfers`
+checkbox to prepare batches of orders for delivery.
+
+In the :guilabel:`Shipping` section, tick the :guilabel:`Delivery Methods` and :guilabel:`Dispatch
+Management System` checkboxes. Doing so allows specific vehicles to be :ref:`set as carriers
+<inventory/shipping_receiving/delivery-method-for-carrier>`.
+
+In the :guilabel:`Warehouse` section, tick the :guilabel:`Storage Locations` checkbox to assign
+specific locations in the warehouse as loading zones for delivery trucks.
+
+.. _inventory/shipping_receiving/delivery-method-for-carrier:
+
+Delivery method
+---------------
+
+Next, assign each delivery vehicle as a :guilabel:`Carrier` by configuring a delivery method.
+
+To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Delivery
+Methods`. Select an existing delivery method, or click :guilabel:`New`.
+
+.. seealso::
+   :doc:`Configure delivery method <../setup_configuration>`
+
+On the :guilabel:`Delivery Method` form, enter a name for the delivery method. It is recommended to
+use identifying information, such as the vehicle description and license plate number, such as
+`Truck 123-ABCD`.
+
+Since the delivery methods are managed internally, set the :guilabel:`Provider` to either
+:guilabel:`Fixed Price` or :guilabel:`Based on Rules`. For more information about how shipping
+prices are calculated, refer to the :doc:`Delivery method <../setup_configuration>` article.
+
+Next, set a :guilabel:`Delivery Product`, which is the product that shows up as the customer's
+:ref:`delivery charge <inventory/shipping_receiving/delivery-product>` on the sales order or
+invoice.
+
+Optionally, in the :guilabel:`Availability` tab, set the :guilabel:`Countries`, :guilabel:`States`,
+or :guilabel:`Zip Prefixes` to limit the range of local delivery.
+
+.. figure:: dispatch/delivery-method.png
+   :alt: Delivery method form.
+
+   Example delivery method, with the :guilabel:`Zip Prefixes` set to San Francisco's zip code.
+
+.. _inventory/shipping_receiving/docks:
+
+Dock locations
+--------------
+
+Each loading dock must have a dedicated location. To create or configure dock locations, go to
+:menuselection:`Inventory app --> Configuration --> Locations`.
```

Comment:
```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouse Management: Locations`.
```

Reply:
Hi @ksc-odoo !

According to the [content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#lists),  it seems that section names are no longer included when navigating through menu items, and we're now listing only the items themselves. Could you confirm if this is correct?

cc: @samueljlieber